sdk/kusto/Azure.ResourceManager.Kusto/tests/Scenario/KustoClusterTests.cs

```diff
@@ -77,12 +77,14 @@ public async Task ClusterTests()
                 ValidateCluster
             );
 
-            await ClusterResourceTests(clusterCollection, clusterName);
+            await ClusterStopStartTests(clusterCollection, clusterName);
```

Comment:
Consider to put the common logic to `Setup` method and split this into multiple tests, so that it would be easier to see which test is failing.

Reply:
I prefer to keep it that way to make it short in terms of time execution.
Cluster create \ update \ and delete operations can take 15 minutes each.

If I split the tests, I will add almost 1 more hour to test recording